maven-plugin-tools-annotations/src/main/java/org/apache/maven/tools/plugin/extractor/annotations/datamodel/ParameterAnnotationContent.java

```diff
@@ -48,20 +48,24 @@
 
     private String className;
 
+    private boolean methodSource;
+
     private final List<String> typeParameters;
 
-    public ParameterAnnotationContent( String fieldName, String className, List<String> typeParameters )
+    public ParameterAnnotationContent( String fieldName, String className, List<String> typeParameters,
+                                       boolean methodSource )
     {
         super( fieldName );
         this.className = className;
         this.typeParameters = typeParameters;
+        this.methodSource = methodSource;
     }
 
     public ParameterAnnotationContent( String fieldName, String alias, String property, String defaultValue,
                                        boolean required, boolean readonly, String className,
-                                       List<String> typeParameters )
+                                       List<String> typeParameters, boolean methodSource )
```

Comment:
maybe `annotationOnMethod` or `methodAnnotation`?

Reply:
change to `annotationOnMethod`